x-pack/plugins/ml/server/models/data_recognizer/modules/sample_datasets_flights/ml/mean_delay_by_dest_weather.json

```diff
@@ -0,0 +1,39 @@
+
+{
+  "groups": [],
+  "description": "Kibana Sample Datasets Flights : detect anomalies in mean flight delay",
```

Comment:
Should use sentence case, and for consistency with other modules - Kibana sample flights data: Detect anomalies in mean flight delay`